File: 2022年11月/01 1662. 检查两个字符串数组是否相等/arrayStringsAreEqual.py
class Solution(object):
    def arrayStringsAreEqual(self, word1, word2):
        """
        :type word1: List[str]
        :type word2: List[str]
        :rtype: bool
        执行用时：16 ms, 在所有 Python 提交中击败了75.93%的用户
        内存消耗：13 MB, 在所有 Python 提交中击败了68.52%的用户
        """
        return "".join(word1) == "".join(word2)

File: src/2022年11月/01 1662. 检查两个字符串数组是否相等/arrayStringsAreEqual.js
/**
 * @param {string[]} word1
 * @param {string[]} word2
 * @return {boolean}
 * 执行用时：64 ms, 在所有 JavaScript 提交中击败了45.45%的用户
 * 内存消耗：41.1 MB, 在所有 JavaScript 提交中击败了56.82%的用户
 */
var arrayStringsAreEqual = function (word1, word2) {
    return word1.join("") == word2.join("");
};